puma/datadog_checks/puma/puma.py

```diff
@@ -0,0 +1,83 @@
+import simplejson as json
+from six.moves.urllib.parse import urlparse
+
+from datadog_checks.base import AgentCheck, ConfigurationError
+
+METRICS = [
+    ('booted_workers', 'booted_workers', AgentCheck.gauge),
+    ('workers', 'workers', AgentCheck.gauge),
+    ('backlog', 'backlog', AgentCheck.gauge),
+    ('max_threads', 'max_threads', AgentCheck.gauge),
+    ('pool_capacity', 'pool_capacity', AgentCheck.gauge),
+    ('running', 'running', AgentCheck.gauge),
+    ('requests_count', 'requests_count', AgentCheck.gauge),
+]
+
+
+class PumaCheck(AgentCheck):
+    def check(self, instance):
+        if 'control_url' not in instance:
+            raise ConfigurationError('Puma instance missing "control_url" value')
+
+        tags = instance.get('tags', [])
+        response, content_type, version = self._perform_service_check(instance, instance.get('control_url'))
+        metrics = self._extract_metrics(response)
```

Comment:
```suggestion
        control_url = instance.get('control_url')
        if control_url is None:
            raise ConfigurationError('Puma instance missing "control_url" value')

        tags = instance.get('tags', [])
        response, content_type, version = self._perform_service_check(instance, control_url)
        metrics = self._extract_metrics(response)
```

Let's set `control_url` to a variable as it's used more than once